app/javascript/forms/mappers/componentMapper.jsx

```diff
@@ -21,6 +23,8 @@ const mapper = {
   'tree-selector': TreeViewSelector,
   [componentTypes.SELECT]: Select,
   'multi-select': MultiSelectWithSelectAll,
+  'font-icon-picker': FontIconPicker,
+  'font-icon-picker-new': FontIconPickerNew, // used for react form pages
```

Comment:
same comment as above, please change file name.

Reply:
Changed to **FontIconPickerDdf**